api/taskcluster/webhook_test.go

```diff
@@ -90,19 +91,19 @@ func TestIsOnMaster(t *testing.T) {
 
 func TestExtractTaskGroupID(t *testing.T) {
 	t.Run("Status", func(t *testing.T) {
-		group, task := extractTaskGroupID("https://tools.taskcluster.net/task-group-inspector/#/Y4rnZeqDRXGiRNiqxT5Qeg")
+		group, task := extractTaskGroupID("https://tc.example.com/task-group-inspector/#/Y4rnZeqDRXGiRNiqxT5Qeg")
```

Comment:
Where's `tc.example.com` from? Does `tcurls.API` default to this URL when the root URL is empty?

Reply:
In this case, it's just test input to the function, and a way of taking out the soon-to-be-defunct hostname.  Elsewhere in the tests, I use it as a testing rootURL.